packages/compass-e2e-tests/tests/instance-databases-tab.test.ts

```diff
@@ -172,10 +178,11 @@ describe('Instance databases tab', function () {
       await mongoClient.close();
     }
 
-    // looks like if you refresh too fast the database appears in the list but
-    // the stats never load, so just pause for bit first
-    await browser.pause(1000);
-    await browser.clickVisible(Selectors.InstanceRefreshDatabaseButton);
+    // Refresh again and the database card should disappear.
+    await browser.clickVisible(Selectors.InstanceRefreshDatabaseButton, {
```

Comment:
Sometimes the card goes away. This is now the only time we hit the refresh button so the only thing I can think of is that the initial load is still somehow in progress by the time we hit the refresh button.